src/order/zorn.lean

```diff
@@ -272,6 +272,12 @@ begin
         (λ ht, h t₁ ht₁ c₁ hc₁ c₂ (ht hc₂) hneq) }
 end
 
+lemma chain_empty : chain ∅ :=
+chain_chain_closure chain_closure_empty
+
+lemma chain_of_subsingleton (hc : set.subsingleton c) : chain c :=
```

Comment:
This allows dot notation and can't be confused with `list.chain` because we can't coerce a set to a list.
```suggestion
lemma _root_.set.subsingleton.chain (hc : set.subsingleton c) : chain c :=
```

Reply:
`to_` is used for instances, but not really for predicates.

Reply:
Well it's used as `linear_map.to_add_monoid_hom`, but maybe you're right that the convention is to only use it for data.

Reply:
I'm conflicted on this haha, don't really have a preference